go/cmd/dolt/commands/sql.go

```diff
@@ -441,7 +441,7 @@ func processQuery(ctx context.Context, query string, se *sqlEngine) error {
 	case *sqlparser.Select, *sqlparser.OtherRead, *sqlparser.Insert, *sqlparser.Update, *sqlparser.Show:
 		sqlSch, rowIter, err := se.query(ctx, query)
 		if err == nil {
-			err = prettyPrintResults(ctx, se.db.Root().VRW().Format(), sqlSch, rowIter)
+			err = prettyPrintResults(ctx, se.sdb.Root().VRW().Format(), sqlSch, rowIter)
```

Comment:
rather than getting the VRW from the root se.ddb.ValueReadWriter().Format() seems better, but better still would be to add a Format method to DoltDB and call that directly here.